### PR TITLE
Update `Featured Artist playlists`

### PR DESCRIPTION
**Pull request by pishifat**
prereq for next fa release

### CODE REVIEW

File: wiki/People/Featured_Artists/Featured_Artist_playlists/en.md [hunk 1]
```diff
@@ -34,3 +142,21 @@ Leaders of each Featured Artist playlist are rewarded with mode-specific profile
 - 15 allowed attempts across all playlist items
 - [Hidden (HD)](/wiki/Gameplay/Game_modifier/Hidden), [Hard Rock (HR)](/wiki/Gameplay/Game_modifier/Hard_Rock), [No Fail (NF)](/wiki/Gameplay/Game_modifier/No_Fail) and [Mirror (MR)](/wiki/Gameplay/Game_modifier/Mirror) freemod options
 - Prizes rewarded to top 0.5% of players in each playlist
+
+### September 2022
+
+- Top 8 beatmaps from the [triangles Beatmapping Contest](/wiki/en/Contests/triangles_Beatmapping_Contest) with separate prizing
+- Only applicable to osu! ruleset
```
Comment (Walavouchey) [inline wiki/People/Featured_Artists/Featured_Artist_playlists/en.md:149]: wiki uses "game mode". when switching terminology, it'll be done all at once

```suggestion
- Only applicable to osu! game mode
```

**Review by Walavouchey — DISMISSED** [review]

File: wiki/People/Featured_Artists/Featured_Artist_playlists/en.md [hunk 2]
```diff
@@ -34,3 +142,21 @@ Leaders of each Featured Artist playlist are rewarded with mode-specific profile
 - 15 allowed attempts across all playlist items
 - [Hidden (HD)](/wiki/Gameplay/Game_modifier/Hidden), [Hard Rock (HR)](/wiki/Gameplay/Game_modifier/Hard_Rock), [No Fail (NF)](/wiki/Gameplay/Game_modifier/No_Fail) and [Mirror (MR)](/wiki/Gameplay/Game_modifier/Mirror) freemod options
 - Prizes rewarded to top 0.5% of players in each playlist
+
+### September 2022
+
+- Top 8 beatmaps from the [triangles Beatmapping Contest](/wiki/en/Contests/triangles_Beatmapping_Contest) with separate prizing
```
Comment (Walavouchey) [inline wiki/People/Featured_Artists/Featured_Artist_playlists/en.md:148]:
```suggestion
- Top 8 beatmaps from the [triangles Beatmapping Contest](/wiki/Contests/triangles_Beatmapping_Contest) with separate prizing
```

File: wiki/People/Featured_Artists/Featured_Artist_playlists/en.md [hunk 3]
```diff
@@ -34,3 +142,21 @@ Leaders of each Featured Artist playlist are rewarded with mode-specific profile
...
+- Only applicable to osu! ruleset
+
+### March 2023
+
+- Top 8 beatmaps from the [Twin Trials Contest](/wiki/en/Contests/Twin_Trials_Contest) with separate prizing
```
Comment (Walavouchey) [inline wiki/People/Featured_Artists/Featured_Artist_playlists/en.md:153]:
```suggestion
- Top 8 beatmaps from the [Twin Trials Contest](/wiki/Contests/Twin_Trials_Contest) with separate prizing
```

File: wiki/People/Featured_Artists/Featured_Artist_playlists/en.md [hunk 4]
```diff
@@ -15,10 +15,118 @@ Access the playlists via the **play** menu on osu!(lazer) and filtering by **Fea
...
 ## Prizes
 
-Leaders of each Featured Artist playlist are rewarded with mode-specific profile badges. Exact criteria for leaders varies per playlist.
+Leaders of each Featured Artist playlist are rewarded with mode-specific profile badges.
 
 ![Featured Artist profile badge](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!.png) ![Featured Artist profile badge](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!taiko.png) ![Featured Artist profile badge](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!catch.png) ![Featured Artist profile badge](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!mania.png)
 
+### Cumulative leaderboard
+
+As of June 2023, prizes are awarded to any user who achieves 10 points in a **cumulative leaderboard** spanning Featured Artist playlists. Scores in different Featured Artist playlists carry over between playlists, excluding playlists with separate prizes, like [September 2022](#september-2022) or [March 2023](#march-2023).
```
[the remark this answers is not in the document]
Comment (pishifat) [reply wiki/People/Featured_Artists/Featured_Artist_playlists/en.md:24]: they're in the wiki articles for triangles/twintrials. i dont think it's worth explaining in this article

[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/People/Featured_Artists/Featured_Artist_playlists/en.md:24]: i mean from this cumulative leaderboard. the wording suggests it leads to a separate prize other than what's mentioned above (mode-specific profile badges)

[the remark this answers is not in the document]
Comment (pishifat) [reply wiki/People/Featured_Artists/Featured_Artist_playlists/en.md]: i made it as straightforward as possible, hope this works

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**